src/main/java/commands/Command.java

```diff
@@ -0,0 +1,30 @@
+package commands;
+
+import functions.*;
```

Comment:
Perhaps you can import the specific classes that you are using in this file instead of using the "*" notation?